README.md

```diff
@@ -0,0 +1,19 @@
+toggle-gifs
+===========
+Start/stop GIF animations through a keyboard shortcut or by clicking them. You can also restart animations from the beginning, or disable animations by default.
+
+
+Development
+-----------
+Requirement on OS level: `yarn` - https://yarnpkg.com/
+
+### Commands
++ `install`     - install all packages
++ `prettier`    - run prettier on all *.js files
++ `eslint`      - run eslint on all *.js files with `--fix` option
++ `test`        - run tests once
++ `tdd`         - run tests and watch for file changes
++ `lint`        - run `web-ext lint` in ./extension
++ `dev:web`     - run webpack in development mode and watch for file changes
++ `dev:ext`     - run web-ext with `./example/gif.html` test page
++ `start`       - run development environment (`dev:web` + `dev:ext`)
```

Comment:
Worth noting here that `yarn install` needs to be run before anything works, and that you may need to press ctrl+shift+r to refresh the testing page after the addon has auto-reloaded (because otherwise no new web requests are made, due to the image cache).

Some command for building a complete XPI will be needed later (and would actually be useful for me to evaluate how much code gets included in the addon, and what gets rewritten how)